packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -287,7 +287,7 @@ function isInsertionPoint( targetToCheck, ownerDocument ) {
 	return !! (
 		defaultView &&
 		targetToCheck instanceof defaultView.HTMLElement &&
-		targetToCheck.dataset.isInsertionPoint
+		targetToCheck.closest( '[data-is-insertion-point]' )
```

Comment:
I assume the change was needed because sometimes the target (the element receiving the drag event) is a child of the insertion point element? Maybe the text or a nested element?

Just wanted to check as it stood out.

Reply:
Yep, there's a child `div` in the insertion point that has the text `DropPattern`, so this function would return `false` for it, even though it's part of the insertion point.

Reply:
Good catch 👍